### PR TITLE
dash/docs-kpi-component

### PR DESCRIPTION
**Pull request by sebastianbochan**
Added KPI Component docs as extra article.

### CODE REVIEW

File: docs/dashboards/kpi-component.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
+KPI Component
+===
+
+The KPIComponent allows you to visualize key performance indicators.
+
+<iframe style="width: 100%; height: 470px; border: none;" src=https://www.highcharts.com/samples/embed/dashboards/components/component-kpi allow="fullscreen"></iframe>
+
+## How to start
+1. The KPIComponent is included in the Dashboard package, so nothing apart from the basic dashboards package needs to be imported.
```
Comment (pawellysy) [inline docs/dashboards/kpi-component.md:9]: KPI is not included in the main Dashboards package. It is registered in the HighchartsPlugin

File: docs/dashboards/kpi-component.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+2. Define a cell using a unique identifier for example cell: `dashboard-col-0`.
```
Comment (pawellysy) [inline docs/dashboards/kpi-component.md:11]:
```suggestion
2. Define a cell using a unique identifier for example `cell: dashboard-col-0`.
```

File: docs/dashboards/kpi-component.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+You can find more information how to create a layout in dashboard [here](https://www.highcharts.com/docs/dashboards/your-first-dashboard).
+
+3. The last thing that you have to do is to specify the `type: 'KPI'` in the component’s config and that’s it. See the full example below.
```
Comment (pawellysy) [inline docs/dashboards/kpi-component.md:15]: You have to also add the `value: <value>` to display anything, so I wouldn't write it like that.

File: docs/dashboards/kpi-component.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+        chartOptions: {
+          series: [{
+              data: [734, 244, 685, 250, 920, 320, 200, 150]
+          }]
+        }
+    }]
+});
+```
+
+## API options
```
Comment (pawellysy) [inline docs/dashboards/kpi-component.md:90]: The KPI components also accept the connector, so I'd add section about connecting the dataSource as well.

**Review by pawellysy — CHANGES_REQUESTED** [review]

File: docs/dashboards/kpi-component.md (hunk 1 above)
Comment (sebastianbochan) [reply docs/dashboards/kpi-component.md:9]: The bottom line here is that you do not need to include an extra module for the KPI, so it's in the core of the Dashboards.

File: docs/dashboards/kpi-component.md (hunk 2 above)
Comment (sebastianbochan) [reply docs/dashboards/kpi-component.md:11]: Missing quotes, so I will commit it on my own.

**Review by pawellysy — CHANGES_REQUESTED** [review]

**Review by pawellysy — APPROVED** [review]

File: docs/dashboards/kpi-component.md [hunk 5]
```diff
@@ -0,0 +1,137 @@
+KPI Component
+===
+
+The KPIComponent allows you to visualize key performance indicators.
```
Comment (karolkolodziej) [inline docs/dashboards/kpi-component.md:4]: I think it's worth mentioning that can be represented by value and you can show sth on the chart.

**Review by karolkolodziej — CHANGES_REQUESTED** [review]
Please also describe:
- sync
- KPI value format

**Review by karolkolodziej — APPROVED** [review]

**Review by Vazonik — APPROVED** [review]

**Pull request merged**